### PR TITLE
BrowseProfiles

### PR DESCRIPTION
**Pull request by dej-code**
Completed functionality for the BrowseProfiles section

### CODE REVIEW

File: src/layout/ProfileCard.jsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import React, { useRef, useState, useEffect } from 'react';
+import { useSpring, animated } from '@react-spring/web';
+import '../styles/ProfileCard.css';
+
```
Comment (beefchief-code) [inline src/layout/ProfileCard.jsx:4]: check capitalization on profileCard.css - causing build failure

**Review by wangbertha — CHANGES_REQUESTED** [review]
BrowseProfiles looks great, just need to clear out the extra routing files/code:

- remove LandingPage.jsx, Home.jsx
- remove router variable from App.jsx

File: src/layout/ProfileCard.jsx (hunk 1 above)
Comment (dej-code) [reply src/layout/ProfileCard.jsx:4]: resolved

**Review by dej-code — COMMENTED** [review]
resolved

**Review by beefchief-code — APPROVED** [review]
looks good! build is working

**Review by wangbertha — APPROVED** [review]

**Pull request merged**